cpp/oneapi/dal/backend/primitives/optimizers/cg_solver_dpc.cpp

```diff
@@ -19,21 +19,23 @@
 #include "oneapi/dal/backend/primitives/blas/gemv.hpp"
 #include "oneapi/dal/backend/primitives/element_wise.hpp"
 #include "oneapi/dal/detail/error_messages.hpp"
+#include "oneapi/dal/detail/profiler.hpp"
 
 namespace oneapi::dal::backend::primitives {
 
 template <typename Float>
-sycl::event cg_solve(sycl::queue& queue,
-                     base_matrix_operator<Float>& mul_operator,
-                     const ndview<Float, 1>& b,
-                     ndview<Float, 1>& x,
-                     ndview<Float, 1>& residual,
-                     ndview<Float, 1>& conj_vector,
-                     ndview<Float, 1>& buffer,
-                     Float tol,
-                     Float atol,
-                     std::int64_t maxiter,
-                     const event_vector& deps) {
+std::pair<sycl::event, std::int64_t> cg_solve(sycl::queue& queue,
```

Comment:
its not critical, but for the most part of oneDAL functions we firstly return values, and after that events

Reply:
At the beginning this function was only returning sycl::event, then num_iter appeared as additional return result